src/backend/core/models.py

```diff
@@ -395,9 +444,26 @@ class Document(BaseModel):
         blank=True,
         null=True,
     )
+    duplicated_from = models.ForeignKey(
```

Comment:
Does this column have a purpose in the system or is it just to get some information ?

Reply:
For the moment just to keep the information as it seems legit. It is not being used for display at the moment.